beacon-chain/params/config.go

```diff
@@ -5,8 +5,8 @@ const (
 	AttesterCount = 32
 	// AttesterReward determines how much ETH attesters get for performing their duty.
 	AttesterReward = 1
-	// EpochLength is the beacon chain epoch length in blocks.
-	EpochLength = 5
+	// EpochLength is the beacon chain epoch length in slots.
```

Comment:
What are slots?

Reply:
slot would refer to a period of 8s, where a proposer will be able to create a block